chapters/ch31.md

```diff
@@ -1 +1,26 @@
-Chương này hiện chưa được dịch, mời bạn đóng góp bản dịch bằng cách [tạo một Pull Request](https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/pulls).
+# Interpreting learning curves: Other cases
```

Comment:
Chỗ này anh không để ý. Số chương cũng nên kèm vào tiêu đề của mỗi chương như bản gốc, thêm một dấu chấm để tách phần số chương và phần chữ. Câu này chuyển thành:
```
# 31. Interpreting ....
```
Em có thể đóng góp một PR khác sửa lại toàn bộ từ đầu được không? Cảm ơn em.

Reply:
Vâng anh, trước mắt em PR phần còn lại  sẽ update thêm số chương, các chương trước đó em sẽ PR sau ạ.